client/src/pages/ToursToday.js

```diff
@@ -72,6 +70,34 @@ const ToursToday = () => {
   );
 };
 
+const sortByPriority = (a, b) => {
+  let indexA = null;
+  let indexB = null;
+  switch (a.status) {
+    case "delivered":
+      indexA = 2;
+      break;
+    case "fetched":
+      indexA = 1;
+      break;
+    default:
+      indexA = 0;
+      break;
+  }
+  switch (b.status) {
+    case "delivered":
+      indexB = 2;
+      break;
+    case "fetched":
+      indexB = 1;
+      break;
+    default:
+      indexB = 0;
+      break;
+  }
+  return indexA - indexB;
+};
+
```

Comment:
This might be a way to shorten this:

```suggestion
const sortByPriority = (a, b) => {
  const options = {
    delivered: 2,
    fetched: 1
  }
  const indexA = options[a.status] ?? 0;
  const indexB = options[b.status] ?? 0;
  return indexA - indexB;
};
```